lib/gh.js

```diff
@@ -193,6 +193,16 @@ GH.projects.setDashAmount = async function (itemId, value) {
   await GH.projects.setFieldValue(itemId, fieldId, value.toString());
 };
 
+GH.projects.setFallbackOwner = async function (itemId, value) {
+  let fieldId = process.env.GITHUB_FALLBACK_OWNER_FIELD;
```

Comment:
TODO: For the future.

Initialize this with the names of the custom fields in GitHub projects and map the human-friendly names from there...